src/sentry/api/endpoints/user_details.py

```diff
@@ -89,3 +97,74 @@ def put(self, request, user):
             return Response(serialize(user, request.user))
 
         return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    @sudo_required
+    def delete(self, request, user):
+        """
+        Delete User Account
+
+        Also removes organizations if they are an owner
+        :pparam string user_id: user id
+        :param list organizations: List of organization ids to remove
+        :auth required:
+        """
+
+        # be strict since we're removing orgs
+        if 'organizations' not in request.DATA or not isinstance(
```

Comment:
you could use a validation/serializer thing to make this a bit more concise (and move the code into that)